engines/stark/ui/menu/saveloadmenu.cpp

```diff
@@ -136,6 +147,8 @@ SaveDataWidget::SaveDataWidget(int slot, Gfx::Driver *gfx, SaveLoadMenuScreen *s
 
 		Graphics::Surface *thumb = metadata.readGameScreenThumbnail(&stream);
 		_texture->update(thumb);
+
+		delete thumb;
 	}
 }
```

Comment:
I feel like putting a `delete save;` here is needed, but it will crash the game. The `StarkEngine::loadGameState()` also doesn't delete the `save`, why is that?